catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -165,14 +172,38 @@ public void setResultShow(List<String> resultShow) {
 
     public void setAttributeAliases(List<String> attributeAliases) {
         this.attributeAliases = attributeAliases.stream()
-                .map(str -> str.split("="))
+                .map(str -> str.split("=", 2))
+                .filter((list) -> {
+                    if (list.length <= 1) {
+                        if (LOGGER.isDebugEnabled()) {
+                            LOGGER.debug("Filtered out invalid attribute/alias: {}", list[0]);
+                        }
+                        return false;
+                    }
+                    return true;
+                })
                 .collect(Collectors.toMap(list -> list[0].trim(), list -> list[1].trim()));
     }
 
     public void setHiddenAttributes(List<String> hiddenAttributes) {
         this.hiddenAttributes = hiddenAttributes;
     }
 
+    public void setAttributeDescriptions(List<String> attributeDescriptions) {
+        this.attributeDescriptions = attributeDescriptions.stream()
+                .map(str -> str.split("=", 2))
+                .filter((list) -> {
+                    if (list.length <= 1) {
+                        if (LOGGER.isDebugEnabled()) {
+                            LOGGER.debug("Filtered out invalid attribute/description: {}", list[0]);
+                        }
+                        return false;
+                    }
+                    return true;
```

Comment:
✏️ If you wish, you could refactor this filter out into a predicate since its almost exactly the same for 192-202 and 177-183